net/xtables-addons/patches/400-deactivate-no-textsearch-bm-error.patch

```diff
@@ -0,0 +1,11 @@
+--- a/extensions/xt_ipp2p.c
++++ b/extensions/xt_ipp2p.c
+@@ -14,7 +14,7 @@
+ #include "xt_ipp2p.h"
+ #include "compat_xtables.h"
+ #if !defined(CONFIG_TEXTSEARCH_BM) && !defined(CONFIG_TEXTSEARCH_BM_MODULE)
+-#	error CONFIG_TEXTSEARCH_BM=y/m is required for ipp2p. Either enable textsearch in your kernel ".config" file, or disable ipp2p in the "mconfig" file in Xtables-addons.
++//#	error CONFIG_TEXTSEARCH_BM=y/m is required for ipp2p. Either enable textsearch in your kernel ".config" file, or disable ipp2p in the "mconfig" file in Xtables-addons.
+ #endif
```

Comment:
I am reluctant to have this patch in our repository. You are already tweaking upstream files, it means that we will have this patch for ages and disabling error message, its not good. For me, it looks like a hack. It will increase maintainership.

Also, I am not going to merge this patch, because it is missing details - who is the author of this patch, there isnt commit subject, description, etc.

Reply:
How else can I indicate that I am the author if not by signed-off-by?

Reply:
https://openwrt.org/submitting-patches#patch_checklist , my proposal is that patches (patch itself), does not have such details why you removed the error. If it is broken, then reach upstream about it, but I dont think that it is broken there.

Reply:
description updated

Reply:
Hm, I am a bit confused with the description.
If xt_ipp2p is not selected, then how is this error even triggered?

Reply:
Well, then why not do what the original commits says:
https://codeberg.org/jengelh/xtables-addons/commit/6b7c62d83ea1b589c3115d959867e0fcbfcf74ea

And remove the modules so they are not built instead of hacking around?

Reply:
I don't know how to dynamically change mconfig based on selected modules.

Reply:
Yeah, this is clearly a hack, I merged it for now but we should really remove the upstream mconf file and generate it on the fly with modules that are selected instead

Reply:
@namiltd Would you please create follow-up issue, so we would not lose of tracking this hack, please? I unresolved this discussion, because it is not resolved and it can not be since it is hack:-)